tests/phpunit/class-wc-stripe-api-test.php

```
@@ -202,6 +202,58 @@ public function mock_unauthorized_response() {
+		try {
+			WC_Stripe_API::request( [], 'test_endpoint', 'POST' );
+		} finally {
+			remove_filter( 'pre_http_request', $capture_filter, 10 );
+		}
+
+		$this->assertIsArray( $captured_args );
+		$this->assertNotTrue(
+			$captured_args['reject_unsafe_urls'] ?? false,
+			'Stripe API POST requests must not set reject_unsafe_urls.'
+		);
+	}
```

PHPMD (2.15.0)

[error] The method test_request_does_not_use_safe_remote_http is not named in camelCase. (undefined)

(CamelCaseMethodName)